packages/cloud-cognitive/src/components/Datagrid/styles/_useInlineEdit.scss

```
@@ -392,6 +392,10 @@ $row-heights: (
   outline-color: $support-01;
 }
 
+.#{$block-class} .#{$carbon-prefix}--text-input:focus {
+  background: $active-ui;
```

I believe @jjennings7 suggested 

```suggestion
  background: $field-01;
```